docs/sdk/push/xamarin-android.md

```diff
@@ -82,6 +82,7 @@ Edit the project's **AndroidManifest.xml** file, then insert the following lines
 
 </application>
 ```
+If your project's **AndroidManifest.xml** already doesn't contain these lines, then there's no additional changes required to the file.
```

Comment:
Minor. `then there's no additional changes` should be `then there are no additional changes`.